<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Small Media Player</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="allinLayout">
    <property name="rightMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="videoLayout"/>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Player</string>
        </property>
        <widget class="QPushButton" name="PlayButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>51</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
        <widget class="QPushButton" name="muteButton">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>40</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QSlider" name="Slider_player_volume">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>30</y>
           <width>22</width>
           <height>41</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_time">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>20</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>00:00:00/00:00:00</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pauseButton">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>20</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>50</y>
           <width>148</width>
           <height>25</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="skipbackwardButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="seekbackwardButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="seekforwardButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="skipforwardButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>50</y>
           <width>54</width>
           <height>24</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QToolButton" name="loopButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="listcontrolButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="fullscreenButton">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>60</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Full</string>
         </property>
        </widget>
        <widget class="QPushButton" name="scrollButton">
         <property name="geometry">
          <rect>
           <x>405</x>
           <y>35</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Media list</string>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>221</width>
         <height>421</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>450</y>
         <width>221</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delButton">
          <property name="text">
           <string>Del</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="looplistButton">
          <property name="text">
           <string>Play_list</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../UIresource.qrc"/>
 </resources>
 <connections/>
</ui>
